Jenkinsfile

```diff
@@ -66,6 +66,27 @@ for(i = 0; i < buildTypes.size(); i++) {
     }
 }
 
+builds.ath = {
+    node("linux") {
+        def fileUri
+        def metadataPath
+        dir("sources") {
+            checkout scm
```

Comment:
Are we guaranteed to get fresh nodes each time in ci.j.i ? I don't remember. If not/in doubt, then a `deleteDir` couldn't hurt I guess.

Reply:
It's not a safe bet that the node will be _entirely_ clean, no.